pkg/controller/v1alpha1/localmodel/controller.go

```diff
@@ -64,8 +64,8 @@ type LocalModelReconciler struct {
 var (
 	ownerKey         = ".metadata.controller"
 	localModelKey    = ".localmodel"
-	apiGVStr         = v1alpha1api.SchemeGroupVersion.String()
-	modelCacheCRName = "LocalModelCache"
+	apiGVStr         = v1alpha1.SchemeGroupVersion.String()
+	modelCacheCRName = "ClusterLocalModel"
```

Comment:
CR name is changed from ClusterLocalModel to LocalModelcache

Reply:
Done